bciers/apps/reporting/src/app/components/finalReview/templates/FieldDisplay.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { LIGHT_GREY_BG_COLOR } from "@bciers/styles";
+
+interface FieldDisplayProps {
+  label: string;
+  value: any;
+  unit?: string;
+  showSeparator?: boolean;
+}
+
+/**
+ * FieldDisplay component is used to display a label, value, and optional unit in a structured format.
+ * The `showSeparator` prop determines whether a horizontal separator should be displayed below the field.
+ */
+export const FieldDisplay: React.FC<FieldDisplayProps> = ({
+  label,
+  value,
+  unit,
+  showSeparator = true,
+}) => {
+  const formattedValue =
+    value === null || value === undefined ? (
+      <span>N/A</span>
+    ) : typeof value === "boolean" ? (
+      <span>{value ? "Yes" : "No"}</span>
+    ) : typeof value === "string" && value.includes(";") ? (
+      value.split(";").map((item, idx) => <div key={idx}>- {item.trim()}</div>)
```

Comment:
the `div` here might have some weird side-effects with flex and text management. It didn't work by just having a string separated by " - " ?